### ISSUE TITLE
Add convenience property for ion.transitions.wavelength[~ion.transitions.is_twophoton]

### ISSUE THREAD

**Issue by dstansby** [wtbarnes/fiasco#88]
`ion.transitions.wavelength[~ion.transitions.is_twophoton]` is the way to get the correspond wavelengths for the contribution function, but this is a bit cumbursome. It would be nice if `Ion` had a property that returned this. Not sure what it should be called though...

**Comment by wtbarnes** [comment]
What do we think a good name for this would be? Should it be a property on `ion.transitions` or `ion`? `transitions_non_two_photon` sounds way too wordy...
